core/src/wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.h

```diff
@@ -109,7 +111,10 @@ namespace ledger {
             std::vector<uint8_t> serializeOutputs() override;
 
             int32_t getVersion() override;
+            
+            void setSignatures(const std::vector<api::BitcoinLikeSignature> & signatures) override;
```

Comment:
Please provide documentation about what that function does. Especially, given the current implementation, is it idempotent?